plugins/auth-backend/config.d.ts

```diff
@@ -152,6 +152,8 @@ export interface Config {
           clientSecret: string;
           callbackUrl?: string;
           metadataUrl: string;
+          tokenEndpointAuthMethod?: string;
+          tokenSignedResponseAlg?: string;
```

Comment:
Updated to match config here: https://github.com/backstage/backstage/blob/1a14adeb574f7846b6d5f5ef555a57dea403e03a/plugins/auth-backend/src/providers/oidc/provider.ts#L239-L244